ddl/ddl_api.go

```diff
@@ -965,10 +966,10 @@ func (d *ddl) CreateTable(ctx sessionctx.Context, s *ast.CreateTableStmt) (err e
 
 	// table exists, but if_not_exists flags is true, so we ignore this error.
 	if infoschema.ErrTableExists.Equal(err) && s.IfNotExists {
-		return nil
+		return tbInfo.ID, nil
```

Comment:
If table exists, we need to execute the insertion of create-select?

Reply:
Of course we shouldn't, I will fix this, thanks!